cmake/BLTMacros.cmake

```diff
@@ -656,5 +656,10 @@ macro(blt_append_custom_compiler_flag)
       set (${arg_FLAGS_VAR} "${${arg_FLAGS_VAR}} ${arg_DEFAULT} ")
    endif()   
 
+   # append any NVCC flags to CUDA_NVCC_FLAGS if CUDA is enabled
+   if ( DEFINED arg_NVCC AND ENABLE_CUDA)
+       list (APPEND CUDA_NVCC_FLAGS ${arg_NVCC})
```

Comment:
I'm not sure this will work exactly how you think. `FindCUDA` sets up the flags as an empty variable in the cache, which causes all kinds of problems.